src/librustc_span/symbol.rs

```diff
@@ -20,6 +20,7 @@ use crate::{Span, DUMMY_SP, GLOBALS};
 #[cfg(test)]
 mod tests;
 
+#[allow(rustc::default_hash_types)]
```

Comment:
This annotation does nothing, it's thrown away during expansion of `symbols!`.

Reply:
Ah I see, thanks!